### ISSUE TITLE
GIBCT Sec. 103 Update - VSP Accessibility Feedback - Design Intent

### ISSUE THREAD

**Issue by 1Copenut** [department-of-veterans-affairs/va.gov-team#7706]
**Feedback framework**

- **❗️ Must** for if the feedback must be applied 
- **⚠️Should** if the feedback is best practice 
- **✔️ Consider** for suggestions/enhancements 

## Must

* Fix the `axe` error where we've got our accordions as `<div>` elements inside an `<ul>`. These are throwing a number of errors that should have been caught by an e2e test. Related issues:
  * https://github.com/department-of-veterans-affairs/va.gov-team/issues/7373
  * https://github.com/department-of-veterans-affairs/va.gov-team/issues/7727
* If we're maintaining the modal window (see my Consider comment) it must not scroll the screen when I open it or close it. We've had a few wrinkles in the past with the screen moving behind modals, so I wanted to raise this for awareness.

## Should

* Always include Certificate of Eligibility (COE) spelling + abbreviation in the description. Just having the text `COE` won't be as clear to screen readers or users not familiar with the term.

## Consider

* Moving the Additional Information glossary definitions to a single glossary page with `#anchor` links. The  visual presentation doesn't accurately depict a modal will open. I'd like to get away from `<button>` elements styled as links as much as possible.

**Comment by amyknox** [comment]
@dneel-bah - "Must" items on this story are dev tasks.

**Comment by dneel-bah** [comment]
- 7727 is resolved and addressed for the 103 enhancement.
- 7373 is ongoing and being included in stories as we go forward
- background movement with modal open is covered in 8313.

**Comment by dneel-bah** [comment]
@1Copenut The must and should items have been addressed and are in staging.  Can this issue be closed?

**Comment by 1Copenut** [comment]
@dneel-bah Yes, this issue can be closed.
